frontend_build/src/extract_$trs.js

```diff
@@ -0,0 +1,72 @@
+var esprima = require('esprima');
+var logging = require('./logging');
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+var path = require('path');
+
+function extract$trs(messageDir, messagesName) {
+  this.messageDir = messageDir;
+  this.messagesName = messagesName;
+}
+
+extract$trs.prototype.apply = function(compiler) {
```

Comment:
Can you give a quick explanation as to what defining a function's `prototype.apply` does?